plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -1805,14 +1989,14 @@ func TestEBSVolumeCountConflicts(t *testing.T) {
 	}{
 		{
 			newPod:       oneVolPod,
-			existingPods: []*v1.Pod{twoVolPod, oneVolPod},
+			existingPods: []*v1.Pod{twoVolPod},
 			maxVols:      4,
```

Comment:
This field should be removed since it's not used anymore

Reply:
fixed